src/lib.rs

```diff
@@ -232,9 +230,9 @@ pub enum ContentType {
 
 impl Copy for ContentType {}
 
-impl Into<i32> for ContentType {
-    fn into(self) -> i32 {
-        match self {
+impl From<ContentType> for i32 {
```

Comment:
I did not know one could implement From my types for types in other crates. Interesting. I wonder why there is no lint for implementing Into then...?